recipes/jh2/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "jh2" %}
+{% set version = "5.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/jh2-{{ version }}.tar.gz
+  sha256: c13d97a3f82a02e6a2a89606f1ffe1771670266dc7746140e00e66c4dad12b14
+
+build:
+  script:
+      - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+      - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+  number: 0
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - {{ compiler('rust') }}
+    - {{ compiler('c') }}
+    - {{ stdlib("c") }}
+    - maturin
+    - cargo-bundle-licenses
+  host:
+    - python
+    - maturin >=1.2,<2.0
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - jh2
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/jawah/h2
+  summary: HTTP/2 State-Machine based protocol implementation
+  license: MIT
+  license_file:
+    - LICENSE
+    - tests/http2-frame-test-case/LICENSE
```

Comment:
I believe you need to list the THIRDPARTY.yml here too.

Reply:
thank you @ocefpaf  - Fixed